docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -174,20 +176,19 @@ TIP: You may override the default value of the `myapp.schema.create` property in
 
 Almost ready!
 To initialize the DB in development mode, we will use the client simple `query` method.
-It returns a `CompletionStage` and thus can be composed to execute queries sequentially:
+It returns an `Uni` and thus can be composed to execute queries sequentially:
 
 [source,java]
 ----
-        client.query("DROP TABLE IF EXISTS fruits")
-                .thenCompose(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Orange')"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Pear')"))
-                .thenCompose(r -> client.query("INSERT INTO fruits (name) VALUES ('Apple')"))
-                .toCompletableFuture()
-                .join();
+ client.query("DROP TABLE IF EXISTS fruits")
+    .onItem().produceUni(r -> client.query("CREATE TABLE fruits (id SERIAL PRIMARY KEY, name TEXT NOT NULL)"))
```

Comment:
Shouldn't we use the same API methods in the quickstart and the doc? (`flatMap` in the quickstart)

Reply:
The quickstart has not been updated. So where did you find the `flatMap`?

Reply:
Sorry, actually `flatMap` is used in the tests (`FruitResource`)

Reply:
oh good catch, yes, it should be consistent (both are equivalent BTW).

Reply:
Fixed.